pkg/utils/deployer.go

```diff
@@ -444,38 +453,43 @@ func applyResourceWithRetry(ctx context.Context, dynamicClient dynamic.Interface
 			return false, nil
 		}
 
-		// try  to update resource
```

Comment:
Better not remove this.

When changes are made from upstream, we need to sync down these changes to child clusters. Here we may need to re-apply those auto-populated fields by downstream child clusters.

Reply:
I did't remove this.